tests/pages/components/conversations/conversation-part-closed.js

```diff
@@ -1,7 +1,15 @@
 import {
-  text
+  hasClass
 } from 'ember-cli-page-object';
 
 export default {
-  title: text('h1')
```

Comment:
Would also love feed back on page object. Think I have everything I need though, at least that I've found I need so far.